sourmash_lib/__init__.py

```diff
@@ -29,7 +31,8 @@ class Estimators(object):
     """
 
     def __init__(self, n=None, ksize=None, protein=False,
-                 with_cardinality=False, track_abundance=False):
+                 with_cardinality=False, track_abundance=False,
+                 max_hash=0, seed=DEFAULT_SEED):
         "Create a new MinHash estimator with size n and k-mer size ksize."
```

Comment:
The docstring could document what each parameter is. For example I am not sure about `protein`, in addition to which it is passed as a value for a named parameter `is_protein` in a nested function call (could be called either `protein` or `is_protein` everywhere then).